### DISCUSSION TITLE
Github Support, nice work time

### DISCUSSION THREAD

**Discussion by wooslow** [community/community#108650] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Maximum frustrated with your support, I bought a subscription so it would be faster but still I've been waiting a month for answers! What to do with such cases? Wait another half a year for an answer? They closed my ticket without saying anything. nice

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @earluv ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
